data.json

```diff
@@ -1,10 +1,10 @@
 {
   "Go": {
-    "Supported": ["go1.10.8", "go1.11.5"],
+    "Supported": ["go1.11.5", "go1.12"],
```

Comment:
@tt the removal of support for 1.10 happens here

Reply:
Sorry, think now I misunderstood your question. This "ends official support" for 1.10 but the buildpack will still work with it, so maybe deprecation is a better word.